src/LCT.PackageIdentifier/NugetProcessor.cs

```diff
@@ -456,28 +467,27 @@ private static void ConvertToCycloneDXModel(List<Component> listComponentForBOM,
                     },
                     new Property()
                     {
-                        Name=Dataconstant.Cdx_IdentifierType,Value="Discovered"
+                        Name=Dataconstant.Cdx_IdentifierType,Value="DISCOVRED"
```

Comment:
correct the  Spelling mistake.Also why in CAPS?

Reply:
Back to normal